.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,22 @@
 <!--
 Thank you for sending the PR! We appreciate you spending the time to work on these changes. 
 
+- [x] Explain the **motivation** for making this change.
+- [x] Provide a **test plan** demonstrating that the code is solid.
+- [x] Match the **code formatting** of the rest of the codebase.
+- [x] Target the `master` branch, NOT a "stable" branch.
+
+## Motivation (required)
+I want to use 'react-native' more comfortable.
+
+What existing problem does the pull request solve?
+---------------------------------------------------------------------
+->in this version, 'button' component couldn't change text size in 'ios'. so, I added button's props 
+"textSizeIOS",be able to change button's text size(May be it's similar 'change button size' in ios.)
+-------------------------------------------------------------
+
+ .  
+=======
```

Comment:
Please revert this file. You are not supposed to change this template file but only modify your PR description.